.github/workflows/check-code.yaml

```diff
@@ -133,3 +133,32 @@ jobs:
             echo "::error::due to bandit violations, return code ${ret}"
             exit 1
           fi
+  check_code_pytest:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 'pypy-3.9'
+          check-latest: true
+      - name: Checkout PyTorch
+        uses: actions/checkout@master
+      - name: Update pip
+        run: pip install --upgrade pip
+      - name: Install pytest
+        run: pip install pytest
+      - name: Checkout plugin current branch
+        uses: actions/checkout@v3
+      - name: Checkout geoips main branch
+        uses: actions/checkout@v3
+        with:
+          repository: NRLMMD-GEOIPS/geoips
+          ref: main
+          token: ${{ secrets.GEOIPS_TOKEN }}
+          path: geoips_dev_utils
+          sparse-checkout: |
+            tests/utils
```

Comment:
I believe this is not needed for pytests, since it is just running pytest in the tests/test_pytest directory, and not using the geoips testing utilities.

Reply:
Address this when implemented (removing from workflow for now)